nautobot/extras/forms/forms.py

```diff
@@ -1499,6 +1511,100 @@ class SecretsGroupFilterForm(NautobotFilterForm):
     q = forms.CharField(required=False, label="Search")
 
 
+#
+# Static Groups
+#
+
+
+class StaticGroupForm(NautobotModelForm):
+    """Generic create/update form for `StaticGroup` objects."""
+
+    content_type = DynamicModelChoiceField(
+        queryset=ContentType.objects.all(),
+        query_params={"feature": "static_groups"},
+    )
+    tenant = DynamicModelChoiceField(queryset=Tenant.objects.all(), required=False)
+
+    class Meta:
+        model = StaticGroup
+        fields = (
+            "name",
+            "content_type",
+            "description",
+            "tenant",
+            "tags",
+        )
+
+
+class StaticGroupFilterForm(NautobotFilterForm):
+    model = StaticGroup
+    q = forms.CharField(required=False, label="Search")
+    content_type = CSVContentTypeField(
```

Comment:
`CSVContentTypeField` should be renamed right? As far as I can tell, this field is only used in filter forms.

Reply:
Yeah, it's a misleading name at this point.